### ISSUE TITLE
Validator: classify setting as tag as ERROR

### ISSUE THREAD

**Issue by rodionos** [kirmanak/Axibase-Charts-Syntax#170]
This must be an ERROR, not a warning.

![image](https://user-images.githubusercontent.com/2098022/43953560-12b6d010-9ca2-11e8-80fa-2d239f932112.png)

**Comment by kirmanak** [comment]
@rodionos Why? Is it error to use a tag named "display"?

**Comment by rodionos** [comment]
The user wanted to hide the series, and instead his config sends `"tags": {"PoolID": "0", "display": "false"}` to the server.

**Comment by kirmanak** [comment]
@rodionos I understand, but what if a user wants to use `display` as a tag, not as a setting? I think the warning is enough, it is not an actual error, the config is still valid.

**Comment by rodionos** [comment]
Lets modify the warning message as follows:

```
{name} setting is interpreted as a series tag and is sent to the server. Remove the setting from the [tags] section or enclose it double-quotes to suppress the warning.
```

**Comment by kirmanak** [comment]
Fixed in #177
